config.toml

```
@@ -64,10 +60,18 @@ home = ["HTML", "RSS"]
     weight = -90
 
   [[menu.main]]
-    identifier = "twitter"
-    name = "Twitter"
-    url = "https://twitter.com/kubewarden"
+    identifier = "bluesky"
+    name = "Bluesky"
+    url = "https://bsky.app/profile/kubewarden.bsky.social"
```

```suggestion
    url = "https://bsky.app/profile/kubewarden.io"
```

Now it is verified :).